### PR TITLE
Update automate-tests.mdx

### PR DESCRIPTION
**Pull request by aravindraveendran7**
Added an additional step of `npm ci` incase of warning while installing playwright package

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3545"></a>

### CODE REVIEW

File: pages/docs/contributing/automated-tests.mdx [hunk 1]
```diff
@@ -7,6 +7,27 @@ We use [Playwright](http://playwright.dev) for writing automated end to end (e2e
...
+                                                                                                                               
+     If your project does not yet depend on Playwright, first install the          
+     applicable npm package (most commonly @playwright/test), and                  
+     then run Playwright's install command to download the browsers:
+
+       npm install @playwright/test                                              
+       npx playwright install  
+
+</p>
+
```
Comment (eddiejaoude) [inline pages/docs/contributing/automated-tests.mdx:30]: Thank you for the suggestion 👍 

I think we should add a step for `npm ci` and then we can skip showing the extra step here

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Note can be created into step 1

File: pages/docs/contributing/automated-tests.mdx (hunk 1 above)
Comment (aravindraveendran7) [reply pages/docs/contributing/automated-tests.mdx:30]: Thank you for the feedback @eddiejaoude .Will update as suggested.

**Comment by aravindraveendran7** [comment]
Closing this PR#3545, as new PR#3809 has been raised for the same issue with respect to the changes recommended in the comments.

**Pull request closed without merging**